tailwind.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  content: [
+    "./src/**/*.{js,jsx,ts,tsx}",
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
```

Comment:
You might want to include [tailwindcss-forms](https://github.com/tailwindlabs/tailwindcss-forms) once you start doing anything form related